src/mumble/MainWindow.cpp

```diff
@@ -428,6 +428,12 @@ void MainWindow::createActions() {
 	gsMoveBack = new GlobalShortcut(this, GlobalShortcutType::MoveBack, tr("Move back", "Global shortcut"));
 	gsMoveBack->setObjectName("gsMoveBack");
 	gsMoveBack->qsWhatsThis = tr("This will move you back into your previous channel");
+
+	gsAdaptivePush = new GlobalShortcut(this, GlobalShortcutType::AdaptivePush, tr("Adaptive Push", "Global Shortcut"));
+	gsAdaptivePush->setObjectName(QLatin1String("gsAdaptivePush"));
```

Comment:
```suggestion
	gsAdaptivePush->setObjectName("gsAdaptivePush");
```
`QLatin1String` shouldn't be necessary (this was changed sometime when we changed buildsystems)

Reply:
`QLatin1String` is simply not enforced anymore. Qt still uses UTF-16 internally and the `QString` construction is going to be slower if we don't give it a hint that it's plain ASCII.